### PR TITLE
Bugfix/fix quality issues

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

File: src/main/java/com/iexec/common/docker/client/DockerClientFactory.java [hunk 1]
```diff
@@ -26,6 +26,8 @@ public abstract class DockerClientFactory {
 
     private static final Map<String, DockerClientInstance> clientsMap = new HashMap<>();
 
+    private DockerClientFactory() {}
```
Comment (zguesmi) [inline src/main/java/com/iexec/common/docker/client/DockerClientFactory.java]: `@NoArgsConstructor(access = AccessLevel.PRIVATE)`

File: src/main/java/com/iexec/common/precompute/PreComputeUtils.java [hunk 2]
```diff
@@ -29,4 +29,7 @@ public class PreComputeUtils {
     public static final String INPUT_FILE_NAMES = "INPUT_FILE_NAMES";
     public static final String IEXEC_PRE_COMPUTE_OUT = "IEXEC_PRE_COMPUTE_OUT";
     // pre-compute writes its output in compute stage's input (/iexec_in).
+
+    private PreComputeUtils() {}
```
Comment (zguesmi) [inline src/main/java/com/iexec/common/precompute/PreComputeUtils.java]: `@NoArgsConstructor(access = AccessLevel.PRIVATE)`

File: src/main/java/com/iexec/common/sdk/order/payload/Order.java [hunk 3]
```diff
@@ -37,5 +37,15 @@ public abstract class Order {
     public boolean isSigned() {
         return sign != null && !sign.isEmpty();
     }
+
+    /**
+     * Converts all characters to Lowercase or returns empty
+     * @param s String to convert
+     * @return the String, converted to lowercase or an empty string
+     */
+    public String toLowerCase(String s) {
```
Comment (zguesmi) [inline src/main/java/com/iexec/common/sdk/order/payload/Order.java:46]: Where is it used?

Comment (jbern0rd) [reply src/main/java/com/iexec/common/sdk/order/payload/Order.java:46]: In the classes from the same package (`com.iexec.common.sdk.order.payload`) inheriting from Order abstract class

File: src/main/java/com/iexec/common/docker/client/DockerClientFactory.java (hunk 1 above)
Comment (jbern0rd) [reply src/main/java/com/iexec/common/docker/client/DockerClientFactory.java]: Do you prefer to use lombok annotation for this ?

File: src/main/java/com/iexec/common/sdk/order/payload/Order.java (hunk 3 above)
Comment (jbern0rd) [reply src/main/java/com/iexec/common/sdk/order/payload/Order.java:46]: I did not find OrderUtils usage in other repositories (blockchain-adapter, result-proxy, sms, core, worker, integration tests, pre/post compute)

File: src/main/java/com/iexec/common/docker/client/DockerClientFactory.java (hunk 1 above)
Comment (zguesmi) [reply src/main/java/com/iexec/common/docker/client/DockerClientFactory.java]: Yes. It reduces boilerplate code.

File: src/main/java/com/iexec/common/precompute/PreComputeUtils.java (hunk 2 above)
Comment (jbern0rd) [reply src/main/java/com/iexec/common/precompute/PreComputeUtils.java]: Fixed in fe571ad

Comment (zguesmi) [reply src/main/java/com/iexec/common/precompute/PreComputeUtils.java]: :pray:

**Review by zguesmi — APPROVED** [review]

**Pull request merged**